src/ninjabrainbot/gui/GUI.java

```diff
@@ -35,58 +35,47 @@
  * Main class for the user interface.
  */
 public class GUI {
-	
-	private MainTextArea mainTextArea;
-	private MainButtonPanel mainButtonPanel;
-	private EnderEyePanel enderEyePanel;
-	
+
+	private final MainTextArea mainTextArea;
+	private final EnderEyePanel enderEyePanel;
+
 	public NinjabrainBotFrame frame;
 	public OptionsFrame optionsFrame;
-	private NotificationsFrame notificationsFrame;
-	private CalibrationPanel calibrationPanel;
-	
-	private Font font;
-	private Font fontLight;
+	private final NotificationsFrame notificationsFrame;
+	private final CalibrationPanel calibrationPanel;
+
 	public Theme theme;
 	public SizePreference size;
-	private ArrayList<ThemedComponent> themedComponents;
-	
+	private final ArrayList<ThemedComponent> themedComponents;
+
 	public Timer autoResetTimer;
-	private static int autoResetDelay = 15 * 60 * 1000;
-	
+	private static final int AUTO_RESET_DELAY = 15 * 60 * 1000;
+
 	public static final int MAX_THROWS = 10;
-	private Calculator calculator;
+	private final Calculator calculator;
 	private ArrayList<Throw> eyeThrows;
 	private ArrayList<Throw> eyeThrowsLast;
 
+	private final Font font;
+
 	public GUI() {
 		theme = Theme.get(Main.preferences.theme.get());
 		size = SizePreference.get(Main.preferences.size.get());
+		font = new Font("雅黑", Font.BOLD, 25);
 		Locale.setDefault(Locale.US);
-		themedComponents = new ArrayList<ThemedComponent>();
+		themedComponents = new ArrayList<>();
 		calculator = new Calculator();
-		eyeThrows = new ArrayList<Throw>();
-		eyeThrowsLast = new ArrayList<Throw>();
-		
+		eyeThrows = new ArrayList<>();
+		eyeThrowsLast = new ArrayList<>();
+
 		Profiler.start("Create frame");
 		frame = new NinjabrainBotFrame(this);
 		notificationsFrame = frame.getNotificationsFrame();
-		
-		// Load fonts
-		Profiler.stopAndStart("Load fonts");
-		try {
-			font = Font.createFont(Font.TRUETYPE_FONT, Main.class.getResourceAsStream("/resources/OpenSans-Regular.ttf"));
-			fontLight = Font.createFont(Font.TRUETYPE_FONT, Main.class.getResourceAsStream("/resources/OpenSans-Light.ttf"));
-			GraphicsEnvironment ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
-			ge.registerFont(font);
-			ge.registerFont(fontLight);
-		} catch (Exception e) {
-			e.printStackTrace();
-		}
-		
```

Comment:
Why is this removed, is it incompatible with chinese characters? The text looks much worse with the default font.

Reply:
Okay I see why you removed it, what I dont understand is why it works fine on certain labels and on others it doesnt. For example almost all labels in the settings menu are fine with, but not in the main window. ![asd](https://i.imgur.com/F6dNLn1.png)
I am not going to remove the better looking font for english users, but I wonder if there is a better solution than to just hard code it and say "use the default font if chinese, otherwise load this font".

Reply:
Because this font cannot display Chinese.
I tried to introduce Chinese fonts, but that would have made the package too big.  Maybe I can make Chinese use the default fonts of the operating system and English use the built-in fonts.

Reply:
Maybe I can try to solve the problem.

Reply:
![image](https://user-images.githubusercontent.com/40592682/147487439-53c36b1c-a4e7-49ba-ae03-cf0944296292.png)
In the code, only the buttons on the Settings page set the font.

Reply:
The font is set everywhere, since ThemedLabels are used everywhere. I think the reason it works in some places is because some labels are enclosed in html tags, which may cause the text to use a default font if a characted doesnt exist in the used font. I would rather not use html tags everywhere though, I think the best solution is to just use different fonts for different languages.

Reply:
ill fix this

Reply:
I need to go to bed because it's midnight here.

Reply:
fixed in 0eeb34d